clang/lib/CodeGen/CGCall.h

```diff
@@ -375,6 +375,11 @@ class ReturnValueSlot {
   bool isExternallyDestructed() const { return IsExternallyDestructed; }
 };
 
+/// If \p F "target-features" are incompatible with the \p TargetOpts features,
```

Comment:
I think this comment needs more elaboration on what this means and what this is for.

Should think about dropping the incompatible functions pass after this too

Reply:
I've updated the comment but I've realized another problem: since it's dropping the functions in the builtin module, the user's module will still have the declaration.

I've updated the test to show this problem.